todo.txt

```diff
@@ -1,6 +0,0 @@
-- define the api use-case and implement
-- prepare the database diagram
-- generate the 42api and test it
-- connect backend to database
-- connect frontend to backend
-- configure vite
```

Comment:
Gotta go fast
![Gotta go fast](https://media.tenor.com/9Eg5sKkoGTcAAAAC/running-away-sonic-the-hedgehog.gif)

Reply:
Don't judge my notes pls :cry: